packages/ui-components/src/chat/message.tsx

```diff
@@ -194,7 +203,11 @@ function MessageComponent(props: MessageProps): JSX.Element {
                     </EditButtons>
                 </div>
             )}
-            {!editMode && <MessageBody>{localMessage.message}</MessageBody>}
+            {!editMode && (
+                <MessageBody>
+                    <ReactMarkdown remarkPlugins={[remarkGfm]}>{processText(localMessage.message)}</ReactMarkdown>
```

Comment:
Rather than using markdown component here directly, could we just pull the dara markdown component implementation to ui-components? It has a bunch of extra styling on top of the defaults to make the markdown look pretty. Can use the updated styling from studio as it's based off of that one.

Reply:
Good point, have added a new component here as discussed :)